unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -170,6 +170,16 @@
     }
   };
 
+  const mockNgdilCredentialOffer = () => {
+    dispatch({
+      type: '[QR Code] Scanned',
+      payload: {
+        form_urlencoded:
+          'openid-credential-offer://?credential_offer_uri=https://api.ngdil-demo.tanglelabs.io/api/offers/creds/u08LmjU8lAcTwx7pLMpy0',
```

Comment:
this url will change so remove, maybe you can wait for my PR I will add steps for dev profile

Reply:
Actually, I would like to get this PR here merged as soon as possible since it contains important UI bug fixes. The PR #117 you mentioned is a great improvement for developer experience, but it's not crucial for most clients who just want to try out the app.

Reply:
Yes but it doesn't fix that the url is not static ;), so maybe you can wait for the other branch to be merged first and then call that function or a similar one.